SSPSolution/SSPSolution/System.h

```diff
@@ -29,6 +32,9 @@ class System
 	InputHandler* m_inputHandler;
 	//this is a physicsHandler
 	PhysicsHandler m_physicsHandler;
+	NetworkModule m_networkModule;
+	char* m_ip = "192.168.1.25";	//Tobias NUC Specific local ip
```

Comment:
Hardcoded local ip which you connect to. @TobbetheStalker need an interface so that the user is able to connect by themself.